js/ui/list/ui.list.base.js

```diff
@@ -319,6 +321,10 @@ export const ListBase = CollectionWidget.inherit({
             return;
         }
 
+        if(!this.option('selectByClick')) {
```

Comment:
I research our code and noticed that target option that we should use is 'selectionByClick' from our CollectionWidget. We understand that 'selectionByClick' should be name 'selectByClick'. Also 'selectionByClick' is private option and we can safety rename it. When this PR will merged let do it. Please, crean